src/Storages/MergeTree/MergeFromLogEntryTask.cpp

```diff
@@ -109,6 +111,18 @@ ReplicatedMergeMutateTaskBase::PrepareResult MergeFromLogEntryTask::prepare()
             };
         }
 
+        int32_t part_metadata_version = source_part_or_covering->getMetadataVersion();
+        if (part_metadata_version > metadata_version)
+        {
+            LOG_DEBUG(log, "Source part metadata version {} is newer then the table metadata version {}. ALTER_METADATA is still in progress.",
```

Comment:
Let's move this below just to make sure that it's actually the source part (and not a covering part)

Reply:
Done.